### DISCUSSION TITLE
Are these wires too long?

### DISCUSSION THREAD

**Discussion by talormanda** [tronikos/esphome-magnetometer-water-gas-meter#42] [Q&A] [answered]
I wanted to use the male pin headers with the female jumper connectors. Is this considered too long? If not, what is considered too long? Using these jumpers would make my project overall easier to implement.
![20241118_180644](https://github.com/user-attachments/assets/9e8f009f-063a-40fc-89bd-0847f3185001)

**Comment by tronikos** [comment, marked as answer]
That should be fine. You will start encountering issues once you are longer than a few meters in which case you will need CAT6.

**Reply by talormanda** [reply]
> That should be fine. You will start encountering issues once you are longer than a few meters in which case you will need CAT6.

Excellent. Thank you. It's much easier to add 2 headers and then jumper the connections. I don't have to worry about outdoor exposure.
